pkg/workloads/console/runner/runner_test.go

```diff
@@ -282,125 +287,275 @@ var _ = Describe("Runner", func() {
 			Expect(err).ToNot(HaveOccurred(), "error while updating console status")
 		}
 
-		Context("When the console is pending", func() {
+		var (
+			timeout time.Duration
+			csl     workloadsv1alpha1.Console
+			readyRb rbacv1.RoleBinding
+		)
+
+		BeforeEach(func() {
+			timeout = 200 * time.Millisecond
+
+			csl = workloadsv1alpha1.Console{
+				ObjectMeta: metav1.ObjectMeta{Name: "test-console"},
+				Spec:       workloadsv1alpha1.ConsoleSpec{User: "test-user"},
+			}
+
+			readyRb = rbacv1.RoleBinding{
+				ObjectMeta: metav1.ObjectMeta{Name: csl.Name},
+				Subjects:   []rbacv1.Subject{{Name: csl.Spec.User}},
+			}
+		})
+
+		Describe("waiting for the console to be ready", func() {
 
 			BeforeEach(func() {
```

Comment:
This style - the space after the describe - is totally novel afaik? Can we cut it back so this file looks like the rest of our tests?

Reply:
I thought it was weird but it's everywhere in the theatre test suites! I thought it was you, and I was trying to blend in. Will cut these lines.

Reply:
Merci beaucoup, it freaks me out